ci/elixir.yml

```diff
@@ -12,10 +12,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
+    - name: Cache Mix Dependencies

```

Comment:
Step names should be "Sentence case" apparently. So say the PR instructions these days :)